src/main/java/teammates/common/datatransfer/attributes/StudentProfileAttributes.java

```diff
@@ -143,6 +124,13 @@ private boolean isMultipleFieldsEmpty() {
 
         return errors;
     }
+    
+    // Helper method: checks if getInvalidityInfoFor* methods returns error and adds it to error list
+    private void getInvalidityInfoErrorAdder(String error, List<String> errors) {
```

Comment:
* Since StudentAttributes, as well as some other *Attributes classes inherit from EntityAttributes, and EntityAttributes requires children to implement the getInvalidityInfo method, all of its children should find this useful. Try moving this method up, and add javadoc for it if you do so?
* getInvalidityInfoErrorAdder -> addError?